tools/pkglint/lib/aws-service-official-names.json

```diff
@@ -0,0 +1,88 @@
+  "AWS::CloudFront": "Amazon CloudFront",
+  "AWS::CloudTrail": "AWS CloudTrail",
+  "AWS::CloudWatch": "Amazon CloudWatch",
+  "AWS::CodeBuild": "AWS CodeBuild",
+  "AWS::CodeCommit": "AWS CodeCommit",
+  "AWS::CodeDeploy": "AWS CodeDeploy",
+  "AWS::CodePipeline": "AWS CodePipeline",
+  "AWS::Cognito": "Amazon Cognito",
+  "AWS::Config": "AWS Config",
+  "AWS::DAX": "Amazon DynamoDB Accelerator",
+  "AWS::DLM": "Amazon Data Lifecycle Manager",
+  "AWS::DMS": "AWS Database Migration Service",
+  "AWS::DataPipeline": "AWS Data Pipeline",
+  "AWS::DirectoryService": "AWS Directory Service",
+  "AWS::DocDB": "Amazon DocumentDB",
+  "AWS::DynamoDB": "Amazon DynamoDB",
+  "AWS::EC2": "Amazon EC2",
+  "AWS::ECR": "Amazon ECR",
+  "AWS::ECS": "Amazon ECS",
+  "AWS::EFS": "Amazon Elastic File System",
+  "AWS::EKS": "Amazon EKS",
+  "AWS::EMR": "Amazon EMR",
+  "AWS::ElastiCache": "Amazon ElastiCache",
+  "AWS::ElasticBeanstalk": "AWS Elastic Beanstalk",
+  "AWS::ElasticLoadBalancing": "Amazon Elastic Load Balancing",
+  "AWS::ElasticLoadBalancingV2": "Amazon Elastic Load Balancing V2",
+  "AWS::Elasticsearch": "Amazon Elasticsearch Service",
+  "AWS::Events": "Amazon CloudWatch Events",
+  "AWS::FSx": "Amazon FSx",
+  "AWS::GameLift": "Amazon GameLift",
+  "AWS::Glue": "AWS Glue",
+  "AWS::Greengrass": "AWS IoT Greengrass",
+  "AWS::GuardDuty": "Amazon GuardDuty",
+  "AWS::IAM": "AWS Identity and Access Management",
+  "AWS::Inspector": "Amazon Inspector",
+  "AWS::IoT1Click": "AWS IoT 1-Click",
+  "AWS::IoT": "AWS IoT",
+  "AWS::IoTAnalytics": "AWS IoT Analytics",
+  "AWS::KMS": "AWS Key Management Service",
+  "AWS::Kinesis": "Amazon Kinesis",
+  "AWS::KinesisAnalytics": "Amazon Kinesis Data Analytics",
+  "AWS::KinesisFirehose": "Amazon Kinesis Data Firehose",
+  "AWS::Lambda": "AWS Lambda",
+  "AWS::Logs": "Amazon CloudWatch Logs",
+  "AWS::MediaStore": "AWS Elemental MediaStore",
+  "AWS::Neptune": "Amazon Neptune",
+  "AWS::OpsWorks": "AWS OpsWorks",
+  "AWS::OpsWorksCM": "AWS OpsWorks CM",
+  "AWS::PinpointEmail": "Amazon Pinpoint Email",
+  "AWS::RAM": "AWS Resource Access Manager",
+  "AWS::RDS": "Amazon Relational Database Service",
+  "AWS::Redshift": "Amazon Redshift",
+  "AWS::RoboMaker": "AWS RoboMaker",
+  "AWS::Route53": "Amazon Route53",
+  "AWS::Route53Resolver": "Amazon Route53 Resolver",
+  "AWS::S3": "Amazon S3",
+  "AWS::SDB": "Amazon SimpleDB",
+  "AWS::SES": "Amazon Simple Email Service",
+  "AWS::SNS": "Amazon Simple Notification Service",
+  "AWS::SQS": "Amazon Simple Queue Service",
+  "AWS::SSM": "AWS Systems Manager",
+  "AWS::SageMaker": "Amazon SageMaker",
+  "AWS::SecretsManager": "AWS Secrets Manager",
+  "AWS::Serverless": "AWS Serverless Application Model",
+  "AWS::ServiceCatalog": "AWS Service Catalog",
+  "AWS::ServiceDiscovery": "Amazon ECS Service Discovery",
+  "AWS::StepFunctions": "AWS Step Functions",
+  "AWS::Transfer": "AWS Transfer for SFTP",
+  "AWS::WAF": "AWS Web Application Firewall",
```

Comment:
How do we handle (the very common case) of a new service being introduced? We need someone to add their name here, no?

Reply:
Yeah - basically if the name is NOT in this list, then the pkglint rule will NOT be able to auto-fix (so build would break). Is it enough if I edit the message to point at the file that has the authoritative list of names?

Reply:
let's start with that